### ISSUE TITLE
Known Bug in Vagrant 1.8.5

### ISSUE THREAD

**Issue by Tset-Noitamotua** [blinkreaction/boot2docker-vagrant#85]
May be it´s better to force vagrant version 1.8.4 installation because of known [issue #7610](https://github.com/mitchellh/vagrant/issues/7610) in vagrant 1.8.5 which will be fixed with [release 1.8.6](https://github.com/mitchellh/vagrant/milestone/18) in September 2016?

**Comment by Tset-Noitamotua** [comment]
vagrant 1.8.4 seems not to be a good option as well because it complains about the newest version of virtualbox (which is 5.1.2) but it demands virtualbox verstion up to 5.0 resulting in error: can´t find virtualbox.

**Comment by Tset-Noitamotua** [comment]
Trying to run this out of the box on a fresh Windows 7 I end up with Vagrant version 1.8.5 and Virtualbox version 5.1.2 and the above mentioned know issue in recent vagrant.

 Having played with this since today early in the morning I am ready to make a break. I hope the world look better tomorow :)))) Appreciate any hints and ideas!

**Comment by Tset-Noitamotua** [comment]
new day ... new try ... 
1. downgraded to vagrant 1.8.4 and manually installed virtualbox 5.0.26
2. ran `bash <(curl -s https://raw.githubusercontent.com/blinkreaction/boot2docker-vagrant/master/scripts/setup.sh)`
3. got some error messages (see screenshots below)
4. `vagrant ssh` into boot2docker
5. works! YEAH!

![image](https://cloud.githubusercontent.com/assets/5635462/17239070/8a171924-5561-11e6-8fb9-e599a030a7b7.png)

![image](https://cloud.githubusercontent.com/assets/5635462/17239071/8ef72ef2-5561-11e6-8504-26dfde7fabbf.png)

![image](https://cloud.githubusercontent.com/assets/5635462/17239073/94fae30c-5561-11e6-94a3-cf18a52d698a.png)

**Comment by achekulaev** [comment]
Hey. Thanks for report. There's indeed a problem with Vagrant at the moment

**Comment by lmakarov** [comment]
@Tset-Noitamotua the error on your 2nd screenshot resulted from a DNS resolution issue.
It might have been one time. Are you still experiencing it?

**Comment by Tset-Noitamotua** [comment]
@lmakarov  you are right ... I tried again from scratch and saw the error only one time on initial start.

It even works`***` from [cmder](http://cmder.net/) (manual start without setup.sh). 

![image](https://cloud.githubusercontent.com/assets/5635462/17434828/fb0c18b8-5b0c-11e6-8bab-3be980e0e8e0.png)

`***` cmder works with some limitations:

![image](https://cloud.githubusercontent.com/assets/5635462/17435427/13a7e45c-5b11-11e6-9869-eab3bdf9b487.png)

cmder complains about sysntax of NET USER and NET SHARE (net.exe)

![image](https://cloud.githubusercontent.com/assets/5635462/17434961/e1d581ee-5b0d-11e6-9017-d67ff55b357e.png)

![image](https://cloud.githubusercontent.com/assets/5635462/17435086/ba3acfa8-5b0e-11e6-9e5d-b91b9ee24f4e.png)

and says `invalid parameter: "||"` when trying to execute icacls.exe 

![image](https://cloud.githubusercontent.com/assets/5635462/17435022/446b7796-5b0e-11e6-9f38-6a11223686e5.png)

Which impact will this have on further use? By chance have an idea for a quick fix for that?

I really like this repo but also really dont like cygwin (and a little less babun). I think it would be a great plus to make it work with cmder, too as cmder is way easier to setup and to use especially for beginners ... like me :))) Moreover if it would work with cmder than it would also work with [ConEmu ](https://github.com/Maximus5/ConEmu) and thus support two of the most popular and actively developed terminal emulators on Windows. May I open a new issue called "Cmder / ConEmu support" and move this commet there?
